content/en/_redirects

```diff
@@ -192,7 +189,8 @@ docs/started/requirements/                    /docs/started/getting-started/
 /docs/guides/*                      /docs/:splat
 /docs/pipelines/concepts/*          /docs/components/pipelines/overview/concepts/:splat
 /docs/pipelines/*                   /docs/components/pipelines/:splat
-/docs/aws/*                         /docs/distributions/aws/:splat
+/docs/aws/*                         /docs/distributions/aws/
+/docs/distributions/aws/*           /docs/distributions/aws/
```

Comment:
Is this line needed? It seems to have same prefix.

Reply:
we need both because 
1] We want to redirect `/docs/distributions/aws/xyz` to `/docs/distributions/aws/`
and 
2] We want to redirect `/docs/aws/xyz` to `/docs/distributions/aws/`

Reply:
> @akartsky the redirects still don't work for `/docs/distributions/aws/test123`

ya weird 

so changed it to `/docs/distributions/aws/*  -> /docs/aws/`
and `/docs/aws/*  -> /docs/distributions/aws/`

now it should definitely work